x-pack/plugin/core/src/main/resources/org/elasticsearch/xpack/deprecation/deprecation-indexing-template.json

```diff
@@ -0,0 +1,17 @@
+{
+  "index_patterns": [".logs-deprecation-elasticsearch"],
```

Comment:
This is a breaking change now, since deprecation logging went into effect in 7.10, and we're changing the index name, are we sure we're okay with the breaking change in a minor release? I'm not sure we should?

Reply:
I yanked it from 7.10 so it hasn't been released yet. This will be the first release in 7.11.

Reply:
Sounds good, I was going based on the PRs but didn't realize it had been pulled, LGTM then.